TombLib/TombLib/Catalogs/TEN Node Catalogs/UI_Hud.lua

```diff
@@ -0,0 +1,563 @@
+-- !Arguments "Number, 20, [ 0 | 360 | 2 ], Rotation\nRange [0 to 360]"
+-- !Arguments "Number, 20, {100}, [ 0 | 1000 | 2 ], Scale X (%)\nRange [0 to 1000]"
+-- !Arguments "Number, 20, {100}, [ 0 | 1000 | 2 ], Scale Y (%)\nRange [0 to 1000]"
+-- !Arguments "Newline, SpriteSlots, 60, Bar sprite sequence object ID, {TEN.Objects.ObjID.CUSTOM_BAR_GRAPHIC}"
+-- !Arguments "Number, 20, [ 0 | 9999 | 0 ], Sprite ID for bar in sprite sequence\nRange[0 to 9999],{1}"
+-- !Arguments "Color, {TEN.Color(255,0,0)}, 20, Color of bar sprite"
+-- !Arguments "Newline, Number, 20, [ -1000 | 1000 | 2 ], Position X (%)\nRange [-1000 to 1000]\nVisible range [0 to 100]"
+-- !Arguments "Number, 20, [ -1000 | 1000 | 2 ], Position Y (%)\nRange [-1000 to 1000]\nVisible range [0 to 100]"
+-- !Arguments "Number, 20, [ 0 | 360 | 2 ], Rotation\nRange [0 to 360]"
+-- !Arguments "Number, 20, {100}, [ 0 | 1000 | 2 ], Scale X (%)\nRange [0 to 1000]"
+-- !Arguments "Number, 20, {100}, [ 0 | 1000 | 2 ], Scale Y (%)\nRange [0 to 1000]"
+-- !Arguments "NewLine, Enumeration, 35, [ Center | Center Top | Center Bottom | Center Left | Center Right | Top Left | Top Right | Bottom Left | Bottom Right ], {3}, Align mode"
+-- !Arguments "Enumeration, 22, [ Fit | Fill | Stretch ], Scale mode"
+-- !Arguments "Enumeration, 28, [ Opaque | Alpha Test | Additive | No Z Test | Subtractive | Wireframe | Exclude | Screen | Lighten | Alphablend ], {9}, Blend mode"
+LevelFuncs.Engine.Node.ConstructLaraBar = function(bartype, showBar, blink, objectIDbg, spriteIDbg, colorbg, posXBG, posYBG, rotBG, scaleXBG, scaleYBG, objectIDbar, spriteIDbar, colorbar, posX, posY, rot, scaleX, scaleY, alignMode, scaleMode, blendMode)
+
+	local operand = bartype+1
+	local alignM = LevelFuncs.Engine.Node.GetDisplaySpriteAlignMode(alignMode)
+	local scaleM = LevelFuncs.Engine.Node.GetDisplaySpriteScaleMode(scaleMode)
+    local blendM = LevelFuncs.Engine.Node.GetBlendMode(blendMode)
+
+	local playerBar = {}
+
+	playerBar.getActionType		= operand
+	playerBar.objectIdBg		= objectIDbg
+	playerBar.spriteIdBg		= spriteIDbg
+	playerBar.colorBg			= colorbg
+	playerBar.posBg				= TEN.Vec2(posXBG, posYBG)
+	playerBar.rotBg				= rotBG
+	playerBar.scaleBg			= TEN.Vec2(scaleXBG, scaleYBG)
+	playerBar.alignModeBg		= alignM
+	playerBar.scaleModeBg		= scaleM
+	playerBar.blendModeBg		= blendM
+	playerBar.objectIdBar		= objectIDbar
+	playerBar.spriteIdBar		= spriteIDbar
+	playerBar.colorBar			= colorbar
+	playerBar.posBar			= TEN.Vec2(posX, posY)
+	playerBar.rot				= rot
+	playerBar.scaleBar			= TEN.Vec2(scaleX, scaleY)
+	playerBar.alignMode			= alignM
+	playerBar.scaleMode			= scaleM
+	playerBar.blendMode			= blendM
+	playerBar.alphaBlendSpeed	= 50
+	playerBar.showBar			= showBar
+	playerBar.blink				= blink
+	playerBar.blinkLimit		= 0.25
+
+	CustomBar.CreatePlayerBar(playerBar)
+
+end
+
+-- !Name "Start/Stop enemy health bars"

```

Comment:
Avoid using technical symbols in names, such as `/`. It should be human readable. Please rephrase as "Start or stop"